zarr-in-practice.ipynb

```diff
@@ -0,0 +1,3003 @@
+{
```

Comment:
<p>I'd suggest adding a second half to the first sentence</p><blockquote>It's important to consolidate metadata because that requires fewer requests to read the dataset metadata when later opening the file.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/cloud-optimized-geospatial-formats-guide/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>